packages/core/src/shared/utilities/workspaceUtils.ts

```diff
@@ -294,6 +295,35 @@ async function filterOutGitignoredFiles(rootPath: string, files: vscode.Uri[]):
     return gitIgnoreFilter.filterFiles(files)
 }
 
+/**
+ * Get the size of the currently sleected workspace and return it in bytes
```

Comment:
typo. please also use punctuation, it's a cheap way to reduce ambiguity.
```suggestion
 * Gets the size of the currently selected workspace and return it in bytes.
 *
```